src/vite.ts

```diff
@@ -56,6 +56,7 @@ async function bundle (nuxt: Nuxt, builder: any) {
           jsxFactory: 'h',
           jsxFragment: 'Fragment'
         },
+        publicDir: resolve(nuxt.options.rootDir, nuxt.options.dir.static),
```

Comment:
https://github.com/nuxt/nuxt.js/blob/40521d0a0d2b3f9308e0c2df033f21b3ede76c7e/packages/server/src/server.js#L93

```suggestion
        publicDir: resolve(nuxt.options.srcDir, nuxt.options.dir.static),
```